### PR TITLE
add Epam GDCR18 event in Kiev <eom>

### PR DESCRIPTION
**Pull request by mmelnik**

### CODE REVIEW

File: _data/events_gdcr2018/Kiev.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR18 in Epam Kiev",
+  "url": "https://events.epam.com/events/coding-dojo-global-day-of-coderetreat",
```
Comment (rradczewski) [inline _data/events_gdcr2018/Kiev.json]: The link redirects to a login page for me. Is this a public event that anyone can join?

Comment (mmelnik) [reply _data/events_gdcr2018/Kiev.json]: Yes, but it seems that link is not opened for "the world" yet.

Comment (rradczewski) [reply _data/events_gdcr2018/Kiev.json]: In that case, I'd suggest we wait until the event page is available. We'll give you a shout out on twitter with the link as well, would be a pity if someone's interested but can't sign up away 😊

Comment (mmelnik) [reply _data/events_gdcr2018/Kiev.json]: Thanks, I'll have to communicate with our admins.

**Comment by rradczewski** [comment]
Any update on this @mmelnik ?

**Pull request closed without merging**